client/src/components/Ticket/index.js

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './style.scss';
 
-function Ticket({ seatNr, type, price }) {
+function Ticket({ seatNr = '3D', price = '10' }) {
+  const [type, setType] = useState('none');
```

Comment:
You should add state to Reservation context and use here context with reducer, perhaps.